server/annotations.go

```diff
@@ -332,10 +393,10 @@ func (s *Service) RemoveAnnotation(w http.ResponseWriter, r *http.Request) {
 }
 
 type updateAnnotationRequest struct {
-	StartTime *time.Time `json:"startTime,omitempty"` // StartTime is the time in rfc3339 milliseconds
-	EndTime   *time.Time `json:"endTime,omitempty"`   // EndTime is the time in rfc3339 milliseconds
-	Text      *string    `json:"text,omitempty"`      // Text is the associated user-facing text describing the annotation
-	Type      *string    `json:"type,omitempty"`      // Type describes the kind of annotation
+	StartTime *time.Time         `json:"startTime,omitempty"` // StartTime is the time in rfc3339 milliseconds
+	EndTime   *time.Time         `json:"endTime,omitempty"`   // EndTime is the time in rfc3339 milliseconds
+	Text      *string            `json:"text,omitempty"`      // Text is the associated user-facing text describing the annotation
+	Tags      *map[string]string `json:"tags"`
```

Comment:
You cam just have this as a `map[string]string`. Maps are already reference types and will be nil by default when instantiated.

Reply:
Cool. I was confused why these types were pointers (as opposed to in the `newAnnotationRequest` struct, which is quite similar) and just cargo culted.